### PR TITLE
[SPARK-7406][Streaming][WebUI] Add tooltips for "Scheduling Delay", "Processing Time" and "Total Delay"

### PR DESCRIPTION
**Pull request by zsxwing**
Screenshots:
screen shot 2015-05-06 at 2 29 03 pm

screen shot 2015-05-06 at 2 24 35 pm

@tdas as we discussed offline

### CODE REVIEW

File: streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala [hunk 1]
```diff
@@ -347,7 +347,7 @@ private[ui] class StreamingPage(parent: StreamingTab)
         <tr>
           <td style="vertical-align: middle;">
             <div style="width: 160px;">
-              <div><strong>Scheduling Delay</strong></div>
+              <div><strong>Scheduling Delay {SparkUIUtils.tooltip("Time taken by Streaming to schedule a batch", "right")}</strong></div>
```
Comment (rxin) [inline streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala]: will this fail style checker?

Comment (zsxwing) [inline streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala]: `scalastyle:off` for the html codes.

**Comment by zsxwing** [comment]
retest this please.

File: streaming/src/main/scala/org/apache/spark/streaming/ui/AllBatchesTable.scala [hunk 2]
```diff
@@ -26,8 +26,10 @@ private[ui] abstract class BatchTableBase(tableId: String) {
   protected def columns: Seq[Node] = {
     <th>Batch Time</th>
       <th>Input Size</th>
-      <th>Scheduling Delay</th>
-      <th>Processing Time</th>
+      <th>Scheduling Delay
+        {SparkUIUtils.tooltip("Time taken by Streaming to schedule a batch", "top")}</th>
```
Comment (rxin) [inline streaming/src/main/scala/org/apache/spark/streaming/ui/AllBatchesTable.scala]: for brevity - do we need to say "by Streaming" ?

Comment (zsxwing) [inline streaming/src/main/scala/org/apache/spark/streaming/ui/AllBatchesTable.scala]: Try to highlight it's not by Spark Core.

Comment (tdas) [inline streaming/src/main/scala/org/apache/spark/streaming/ui/AllBatchesTable.scala]: I think he wanted ot highlight that the delay is in the streaming scheduler and not the DAGScheduler/Taskscheduler. How about "Time taken by streaming scheduler to submit jobs of a batch."

**Comment by zsxwing** [comment]
Updated. How about now?

**Comment by zsxwing** [comment]
retest this please

**Pull request closed without merging**